docs/using_lagoon/docker_images/rabbitmq.md

```diff
@@ -11,16 +11,16 @@ This image is prepared to be used on Lagoon which leverages OpenShift. There are
 - The file `/etc/rabbitmq/definitions.json` is parsed through [envplate](https://github.com/kreuzwerker/envplate) with an container-entrypoint.
 
 ## Included RabbitMQ default schema (definitions.json)
-To enable the support for Mirrored Queues, at least one [`policy`](https://www.rabbitmq.com/parameters.html#policies) must exists.   
-Into the `definitions.json` schema file, are definied the minimal entities to let the container running: virtualhost (vhost), username and password to access management UI, permissions and policies.  
+To enable the support for Mirrored Queues, at least one [`policy`](https://www.rabbitmq.com/parameters.html#policies) must exists.
```

Comment:
the spaces at the end of lines are needed, see https://gist.github.com/shaunlebron/746476e6e7a4d698b373

Reply:
For some reason Visual Studio Code, when I add two blank spaces and save the file, it reverts to version without them. 
First version of this file was written with VIM, and it doesn't scape blank spaces at all.